content/news/XXXX-XX-XX-mixxx-2-5-beta-released.md

```diff
@@ -0,0 +1,491 @@
+title: Mixxx 2.5 beta released
+authors: Evelynne Veys
+tags: 2.5, beta, release announcement
+comments: yes
+status: draft
+
+#### Dear Mixxx-ers
+
+Funny things happen in spring, birds start nesting and lay eggs, and so did the Mixxx developers.
+A bit late for Easter but it is a magnificent giant gift filled with smaller gifts: Mixxx 2.5 beta!
+You can find the details about the new version below.
+
+#### Appeal
+
+Before releasing 2.5 as the next stable release we need it to be tested thoroughly, something we can't do on our own.
+After some time looking at the same release and searching for solutions we might forget miner bugs or problems.
+That's why we need **YOU** to help us. The more people testing this Beta and reporting eventual bugs to the [bugtracker](https://github.com/mixxxdj/mixxx/issues), the faster the beta can be promoted to stable.
+In case you're scared of messing up your neetly configured PC and losing your holy data is your Haloween-nightmare, read the new wiki article [safeguard your Mixxx Data](https://github.com/mixxxdj/mixxx/wiki).
+It explains about making a timestamp-image, backing up your data, tips and tricks.
+Please [join the Mixxx team](https://mixxx.org/get-involved/).
+In Mixxx we trust.
+
+#### Highlights
+
+* In version 2.5 Mixxx steps up from QT 5 to QT 6 [QT-website](https://www.qt.io/product/qt6).
+* Important for all controller mapping contributors: the introduction of the new handling of controller-mappings. Please give us feedback.

```

Comment:
```suggestion
* Important for all controller mapping contributors: the introduction of the custom per-mapping settings . Please give us feedback.
```

Since we don't have strong doc around that, it might be worth to reference the [PR for the only mapping using that feature](https://github.com/mixxxdj/mixxx/pull/12995), so mapping developers can take inspiration from it?